### PR TITLE
chore: add provisioner api version to /buildinfo

### PR DESCRIPTION
**Pull request by aslilac**
This'll be helpful for a feature on the provisioners page that we want to start work on soon

### CODE REVIEW

**Review by Emyrk — APPROVED** [review]

File: site/src/testHelpers/entities.ts [hunk 1]
```diff
@@ -215,6 +215,7 @@ export const MockProxyLatencies: Record<string, ProxyLatencyReport> = {
 
 export const MockBuildInfo: TypesGen.BuildInfoResponse = {
 	agent_api_version: "1.0",
+	provisioner_api_version: "1.1",
```
Comment (f0ssel) [inline site/src/testHelpers/entities.ts:218]: Since current is `2.2` do we want that as the default instead?

**Review by f0ssel — APPROVED** [review]

File: site/src/testHelpers/entities.ts (hunk 1 above)
Comment (Emyrk) [reply site/src/testHelpers/entities.ts:218]: It's 1.1 isn't it?
https://github.com/coder/coder/blob/7518aac879cdcda88aa7f6af6c6841ea6f280ee4/provisionerd/proto/version.go#L5-L14

Comment (aslilac) [reply site/src/testHelpers/entities.ts:218]: `agent_api_version` is the one that's currently 2.2

Comment (Emyrk) [reply site/src/testHelpers/entities.ts:218]: Ah agent api is something else

**Pull request merged**